player/models.py

```diff
@@ -32,3 +34,23 @@ def create(username, password, birth_date, favorite_ball_game):
         player.user.save()
         player.save()
         return player
+
+    def validate_and_save(self, birth_date, favorite_ball_game):
```

Comment:
parameterized now covers 3 cases, 
1. bad date & ball_game
2. bad ball_game good date
3. bad date good ball_game

the test above test for good date & ball_game so that is 4 by now, i belive thoes 4 cases provides full coverage of this method.